python_modules/dagster/dagster/core/storage/alembic/versions/026_convert_start_end_times_format_mysql.py

```diff
@@ -1,23 +1,29 @@
 """convert start end times format
 
 Revision ID: 130b087bc274
-Revises: 17154c80d885
+Revises: b37316bf5584
 Create Date: 2022-02-01 15:21:22.257972
 
 """
+from alembic import op
+
 from dagster.core.storage.migration.utils import (
     add_run_record_start_end_timestamps,
     drop_run_record_start_end_timestamps,
 )
 
 # revision identifiers, used by Alembic.
 revision = "130b087bc274"
-down_revision = "17154c80d885"
+down_revision = "b37316bf5584"
 branch_labels = None
 depends_on = None
 
 
 def upgrade():
+    bind = op.get_bind()
+    if bind.engine.name != "mysql":
```

Comment:
This is a MySQL only migration, where we messed up the start/end due to timestamp shenanigans.  It destructively blows away the run record timestamps, so we do a dialect check here.